src/codegen/helpers.ts

```diff
@@ -1,33 +1,47 @@
+import * as Antelope from '@wharfkit/antelope'
 import {ABI} from '@wharfkit/antelope'
 import * as ts from 'typescript'
 
 import {capitalize} from '../utils'
 
-export const EOSIO_CORE_CLASSES = [
-    'Asset',
-    'Checksum256',
-    'Float64',
-    'Name',
-    'TimePoint',
-    'TimePointSec',
-    'UInt128',
-    'UInt16',
-    'UInt32',
-    'UInt64',
-    'UInt8',
-]
+const EOSIO_CORE_CLASSES: string[] = []
```

Comment:
We should probably rename this variable too 😄 Maybe  to `ANTELOPE_CLASSES`?

Reply:
Totally agreed, didn't even think about that 😂